### PR TITLE
Add Stillsuit when user has Quantized Familiar

### PR DESCRIPTION
**Pull request by Ignose**
Because we cannot predict effects from any other familiar (there's an element of RNG) this is truly the first familiar we can rely on for distillate data. We could probably improve this with a regex to parse the actual distill page, and I may try to do that in the future.

### CODE REVIEW

File: ITEMS.md [hunk 1]
```diff
@@ -104,3 +104,4 @@ The following is a list of currently supported IotMs. Note that because this is
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | Parse actual item% from stillsuit to determine if other familiars would offer higher item drops.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (Pantocyclus) [inline ITEMS.md:107]: The items listed here are sorted by (1) IotMs, (2) IotYs and (3) others, with each subgroup organized by chronological order for ease of reference. This entry should be parked between parka and designer sweatpants.

We should replace quantized familiar with [quantum entangler](https://kol.coldfront.net/thekolwiki/index.php/Quantum_entangler) (with the hyperlink), and I would remove the "Parse actual item% from stillsuit to determine if other familiars would offer higher item drops" for now - it's more of an optimization problem than simply not supporting a feature that the item provides

File: src/tasks/boozedrop.ts [hunk 2]
```diff
@@ -389,6 +400,14 @@ export const BoozeDropQuest: Quest = {
...
         handleCustomPulls("instant_boozeTestPulls", boozeTestMaximizerString);
 
+        if (
+          CommunityService.BoozeDrop.actualCost() > 1 &&
+          StillSuit.distillateModifier("Item Drop") >= 15 &&
+          !get("instant_saveStillsuit", false) &&
+          myInebriety() + 1 < inebrietyLimit()
+        )
+          StillSuit.drinkDistillate();
+
```
Comment (Pantocyclus) [inline src/tasks/boozedrop.ts:410]: Is there a logic behind having it as a task above and once again here? If this is intentional, should we also check for the `Buzzed on Distillate` effect before we distillate further (it's quite unlikely that we can if we distillated earlier)

File: src/tasks/runstart.ts [hunk 3]
```diff
@@ -89,6 +90,14 @@ import { baseOutfit, unbreakableUmbrella } from "../outfit";
...
 import { chooseFamiliar, cookbookbat, melodramedary, sombrero } from "../familiars";
 
+const bestStillsuitFamiliar = have($familiar`quantum entangler`)
+  ? $familiar`quantum entangler`
+  : have($familiar`Gelatinous Cubeling`)
+    ? $familiar`Gelatinous Cubeling`
+    : have($familiar`Blood-Faced Volleyball`)
+      ? $familiar`Blood-Faced Volleyball`
+      : $familiar`Mosquito`;
+
```
Comment (Pantocyclus) [inline src/tasks/runstart.ts:100]: This seems quite unwieldy to add on in the future. We can do something like
```
const bestStillsuitFamiliar = $familiars`quantum entangler, Gelatinous Cubling, Blood-Faced Volleyball, Mosquito`.filter(have)?.at(0) ?? $familiar.none;
```
or the way we do it in `chooseFamiliar()` in `familiar.ts`

File: src/tasks/runstart.ts [hunk 4]
```diff
@@ -145,6 +154,15 @@ export const RunStartQuest: Quest = {
...
     },
+    {
+      name: "Set up Sweatsuit",
+      ready: () => have($item`tiny stillsuit`),
+      completed: () => familiarEquippedEquipment(bestStillsuitFamiliar) === $item`tiny stillsuit`,
+      do: (): void => {
+        equip(bestStillsuitFamiliar, $item`tiny stillsuit`);
+      },
+      limit: { tries: 1 },
+    },
```
Comment (Pantocyclus) [inline src/tasks/runstart.ts:165]: It's hard to imagine a Guy that neither has a volleyball nor mosquito, but we should do our due dilligence to ensure that `bestStillsuitFamiliar` is not `$familiar.none` after we update the variable above

Comment (Ignose) [reply src/tasks/runstart.ts:165]: You cannot not have a mosquito. It's mandatory to ascend once, which is mandatory to enter CS.

**Pull request merged**